client/src/Authorization/Login/Login.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from "react";
+import { Redirect } from "react-router-dom";
+import { authService } from "../_auth-services/authentication";
+import "./Login.css";
+import bookImage from "../../pictures/books.png";
+
+class Login extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            username: "",
+            password: "",
+            submitted: false, 
+            redirect: false
+        };
+        if(authService.authenticateUser() === null) { // If user already logged in, we redirect him to main page
+            this.setState({redirect: true});
+        }
+    }
+
+    loginHandler = async event => {
+        const { username, password } = this.state;
+
+        event.preventDefault();
+        this.setState({ submitted: true });
+        if (!(username && password)) return;
+
+        const response = await authService.login(username, password);
+        if(response.success) {
+            const user = { // object containing user authorisation data
+                name: username,
+                token: response.token
+            }
+            window.localStorage.setItem('user', JSON.stringify(user)); // user data kept in local storage under 'user' key
```

Comment:
just remove comments everywhere. 
'user' key should be stored in a static class.